runtime/bin/tapasco-load-bitstream

```diff
@@ -24,34 +24,42 @@ import os
 import sys
 import subprocess
 from   socket import gethostname
+from   pathlib import Path
 
 parser = argparse.ArgumentParser()
 parser.add_argument('bitstream', help='.bit file to load')
 parser.add_argument('--reload-driver', help='reload driver, if loaded (default: %(default)s)', action='store_true')
 parser.add_argument('--verbose', help='verbose output (default: %(default)s)', action='store_true')
 parser.add_argument('--mode', help='select the operation to be performed (default: %(default)s)', default='normal', choices=['normal', 'program', 'hotplug'])
-args = parser.parse_args() 
+args = parser.parse_args()
 
 if not 'TAPASCO_HOME_RUNTIME' in os.environ:
-        sys.exit('ERROR: Environment variable TAPASCO_HOME_RUNTIME not set. Please source tapasco-setup.sh first or use -E flag to preserve pervious users environment if using sudo.')
-
-if not os.path.exists(args.bitstream):
-	sys.exit('ERROR: {} could not be opened'.format(args.bitstream))
+	sys.exit('ERROR: Environment variable TAPASCO_HOME_RUNTIME not set. Please source tapasco-setup.sh first or use -E flag to preserve pervious users environment if using sudo.')
 
 if not 'TAPASCO_PLATFORM' in os.environ:
 	print('Environment variable TAPASCO_PLATFORM is not set, guessing Platform ...')
 	if gethostname() in ['zed', 'zedboard', 'zc706', 'zynq', 'pynq', 'ultra96v2']:
 		os.environ['TAPASCO_PLATFORM'] = 'zynq'
 	else:
-		os.environ['TAPASCO_PLATFORM'] = 'pcie'
+		hyp_test = Path("/sys/hypervisor/uuid")
```

Comment:
This fails if Tapasco is not run on a hypervisor. There has to be a fallback to PCIe

Reply:
Had better days...